assets/agenda/components/AgendaCoverages.tsx

```diff
@@ -53,7 +53,7 @@ function AgendaCoveragesComponent({item, coverages, wireItems, actions, user, on
                 const deskName = getCoverageDeskName(coverage);
                 const assignedUserEmail = coverage.assigned_user_email;
                 const assignedDeskEmail = coverage.assigned_desk_email;
-                const subject = `Coverage inquiry from NewsPro user: ${item.name || item.slugline}`;
+                const subject = gettext(`Coverage inquiry from ${window.sitename} user: ${item.name || item.slugline}`);
```

Comment:
This will not work as intended when flagging for translation. You will need to pass the dynamic values as a dictionary to the 2nd argument of `gettext` with the key names used in the translation text